### PR TITLE
Update error-templates.md

### PR DESCRIPTION
**Pull request by andyo-tyk**
Added note that it's not possible to use a template for HTTP 404

## Related Issue (if applicable)
https://github.com/TykTechnologies/tyk/issues/3566

## Description
Override messages for HTTP 404 are not supported

### CODE REVIEW

**Review by lghiur — APPROVED** [review]

File: tyk-docs/content/advanced-configuration/error-templates.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ In v2.2 the error handler allowed the use a single JSON template to communicate
 
 As of v2.3 it is possible to use different templates for specific `HTTP error codes`. The `content-type` header of the request is also checked, enabling the usage of different template formats, e.g. an XML template.
 
+Please note that it is not possible to override the default message for HTTP 404 errors, which indicate that the requested resource was not available (e.g. the requested URL does not exist).
```
Comment (dcs3spp) [inline tyk-docs/content/advanced-configuration/error-templates.md]: @andyo-tyk  LGTM. Maybe consider the following suggestion:

```suggestion
Please note that it is not possible to override the default message for HTTP 404 errors. These errors indicate that the requested resource could not be found (e.g. the requested URL does not exist).
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]

**Review by dcs3spp — APPROVED** [review]
lgtm

**Pull request merged**